### PR TITLE
[UPD] update website certificate from superman

### PR DESCRIPTION
**Pull request by xie8899**

### CODE REVIEW

File: website_certificate/README.rst [hunk 1]
```diff
@@ -1,12 +1,10 @@
 .. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
-   :alt: License: AGPL-3
+    :alt: License
 
-===================
 Website Certificate
-===================
+==============
```
Comment (noahzaozao) [inline website_certificate/README.rst]: fix ===

File: website_certificate/__init__.py [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (noahzaozao) [inline website_certificate/__init__.py]: header

File: website_certificate/__openerp__.py [hunk 3]
```diff
@@ -1,20 +1,33 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (noahzaozao) [inline website_certificate/__openerp__.py]: header

File: website_certificate/certificate.py [hunk 4]
```diff
@@ -1,8 +1,25 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (noahzaozao) [inline website_certificate/certificate.py]: header

**Comment by xie8899** [comment]
@noahzaozao @elicoidal @seb-elico Done

**Comment by noahzaozao** [comment]
@seb-elico Please review this PR , thanks

**Comment by seb-elico** [comment]
LGTM

**Pull request merged**